migrations/index.js

```diff
@@ -17,14 +17,11 @@ const main = async function () {
   }
 
   if (dbConfig.host && dbConfig.database) {
-    console.log("Running Migrations");
```

Comment:
This doesn't affect production. So I don't think this needs to be removed at all